tests/unit/base/test_utils.py

```diff
@@ -1,4 +1,7 @@
 import os
+import mock
+import boto3
```

Comment:
@nsjcorps can you re order the imports to follow lexicographic order

Reply:
I have worked on it @Ram81